modules/ossm-uninstalling-cert-manager.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/install/ossm-cert-manager.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-uninstalling-cert-manager_{context}"]
+= Uninstalling Service Mesh with the cert-manager Operator using the Istio-CSR agent
```

Comment:
```suggestion
= Uninstalling Service Mesh with the cert-manager Operator by using the Istio-CSR agent
```
Should "Istio-CSR" be capitalized as shown? In the body text, it's all lowercase.